src/middlewared/middlewared/plugins/iscsi.py

```diff
@@ -606,14 +606,25 @@ class Config:
     @accepts(
         Int('id'),
         Bool('remove', default=False),
+        Bool('validate', default=True),
```

Comment:
Small nit, `validate` feels strange.

Can we use `force` and default to False instead?

Reply:
Sure